addons/filesystem/functions/fnc_device_addDir.sqf

```diff
@@ -0,0 +1,52 @@
+/**
+ * PUBLIC
+ *
+ * Adds a directory to a given Device. Device could be a comupter or a flash drive.
+ *
+ * Arguments:
+ * 1: Device <OBJECT>
+ * 2: Path <STRING>
+ * 3: Owner <STRING>
+ * 4: Permissions <[ARRAY]>
+ *
+ * Results:
+ * Logs if dir already exists; On other errors it throws an exception
+ *
+ * Example:
+ * [_device, "/tmp/new", "root", [[true, true, true], [true, true, true]]] call AE3_filesystem_fnc_device_addDir;
+ *
+ *
+ * Permissions:
+ * [[owner execute, owner read, owner write], [everyone execute, everyone read, everyone write]]
+ */
+
+params ["_device", "_path", "_owner", "_permissions"];
+
+private _filesystem = _device getVariable "AE3_filesystem";
```

Comment:
Get from server

Reply:
I will restrict this function to server-only.